tests/integration/README.md

```diff
@@ -1,16 +1,22 @@
 # Integration Tests
+
 Package for defining integration tests. Currently, there is a setup for API and Orchestrator testing.
 
 ## Run locally
-1) Setup env variables in the .env file
-2) If necessary, run `make connect-orchestrator` to create a tunnel to one orchestrator client VM in GCP
-3) Run `make test`
+
+1. Setup env variables in the `infra/.env` file
+2. If you made changes to the `api` or `envd` protobuf spec, run `make generate` from this folder (and don't forget to generate it in `envd` if changes apply there too).
```

Comment:
If you made changes to the `api` or `envd` protobuf spec, run `make generate` in their respective directories (`packages/api`, `packages/envd`).